addons/website/views/assets.xml

```diff
@@ -75,7 +75,6 @@
     </xpath>
     <xpath expr="//script[last()]" position="after">
         <script type="text/javascript" src="/website/static/src/js/set_view_track.js"/>
-        <script type="text/javascript" src="/website/static/lib/jstz.min.js"/>
```

Comment:
This is landing after some 14.0 databases are already live, so we can delete the inclusion of the files from the assets, but we should still keep 2 empty js files instead of completely deleting them. Otherwise the automatic re-compilation of assets may fail on existing databases: they will have the old assets definitions, but with the new JS files, or absence thereof.
The 2 empty files can then be dropped in the fwd-port.